lib/ClangImporter/ClangModuleDependencyScanner.cpp

```diff
@@ -489,9 +491,15 @@ bool ClangImporter::getHeaderDependencies(
                            ModuleOutputKind MOK) -> std::string {
       return moduleCacheRelativeLookupModuleOutput(MID, MOK, moduleOutputPath);
     };
+    std::unique_ptr<llvm::MemoryBuffer> maybeBuffer;
+    std::optional<llvm::MemoryBufferRef> maybeBufferRef;
+    if (content) {
+      maybeBuffer = llvm::MemoryBuffer::getMemBufferCopy(*content);
+      maybeBufferRef = maybeBuffer->getMemBufferRef();
+    }
     auto dependencies = clangScanningTool.getTranslationUnitDependencies(
         commandLineArgs, workingDir, cache.getAlreadySeenClangModules(),
-        lookupModuleOutput);
+        lookupModuleOutput, /*AdditionalModules=*/{}, maybeBufferRef);
```

Comment:
I see this is added in the corresponding LLVM PR but not used here. What's the thinking behind `AdditionalModules`?